articles/4_flex.md

```diff
@@ -0,0 +1,141 @@
+# Flex
+
+## Displays
+
+A commonly used property in CSS is `display`. This specifies how the element is displayed (if it is shown at all). By default, the display value is either `block` or `inline`, depending on the element (inline elements are things like `a`s, `p`s, `span`s, etc, and block elements are things like `div`s and `form`s)
+
+There is also `display: inline-block`, which treats the element as `inline`, but the element can have a defined width and height, which `inline` elements cannot do on their own.
+
+To hide an element, you set it to `display: none`, this will remove the element from the page. The difference between `display: none` and `opacity: 0` (both of which hide the element) is that changing `opacity: 0` maintains the space the element takes up, whereas `display: none` does not. The element will take up no space.
```

Comment:
I'd mention `visibility: hidden` as an option (basically does the same thing as opacity: 0)